docs/sources/setup-grafana/configure-security/configure-scim-provisioning/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - ../../auth/scim/
+  - ../../enterprise/scim/
+description: Learn how to use SCIM provisioning to synchronize users and groups from your identity provider to Grafana.
+keywords:
+  - grafana
+  - scim
+  - provisioning
+  - user-management
+  - team-management
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Configure SCIM provisioning
+title: Configure SCIM provisioning
+weight: 300
+---
+
+# Configure SCIM provisioning
+
+System for Cross-domain Identity Management (SCIM) is an open standard that allows automated user provisioning and management. With SCIM, you can automate the provisioning of users and groups from your identity provider to Grafana.
+
+> **Note:** Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+
+{{% admonition type="note" %}}
+This feature is behind the `enableSCIM` feature toggle.
+You can enable feature toggles through configuration file or environment variables. See configuration [docs](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles) for details.
+{{% /admonition %}}
+
+## Benefits of SCIM provisioning
+
+SCIM provisioning offers several advantages for managing users and teams in Grafana:
+
+- **Automated user provisioning**: Automatically create, update, and disable users in Grafana when changes occur in your identity provider.
+- **Automated team provisioning**: Automatically create, update, and delete teams in Grafana based on groups in your identity provider, and synchronize team memberships.
+- **Reduced administrative overhead**: Eliminate manual user management tasks and reduce the risk of human error.
+- **Enhanced security**: Automatically disable access when users leave your organization, ensuring proper access control and compliance with security policies.
+
+{{% admonition type="note" %}}
+SCIM provisioning currently works **only with SAML authentication**. Other authentication methods are not supported.
+{{% /admonition %}}
```

Comment:
- [Don't use bold for emphasis](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold)
- Keep documentation timeless by avoiding words like "currently". Currently also implies that this is planned to change. Readers will want to know when this is planned to change.

```suggestion
{{< admonition type="note" >}}
SCIM provisioning only works SAML authentication.
Other authentication methods aren't supported.
{{< /admonition >}}
```